src/objects.h

```diff
@@ -60,18 +65,10 @@
 **  'ARE_INTOBJS' returns 1 if the objects <o1> and <o2> are both (immediate)
 **  integer objects.
 */
-#define ARE_INTOBJS(o1,o2) \
-    ((Int)(o1) & (Int)(o2) & 0x01)
-
-
-/****************************************************************************
-**
-*F  INTOBJ_INT( <i> ) . . . . . . .  convert a C integer to an integer object
-**
-**  'INTOBJ_INT' converts the C integer <i> to an (immediate) integer object.
-*/
-#define INTOBJ_INT(i) \
-    ((Obj)(((UInt)(Int)(i) << 2) + 0x01))
+static inline Int ARE_INTOBJS(Obj o1, Obj o2)
+{
+    return (Int)o1 & (Int)o2 & 0x01;
+}
```

Comment:
This now no longer matches the behavior of `IS_INTOBJ` exactly. Well, of course it does in the current setup; but if we ever add anything with both LSBs set, this function will become incorrect.

I am not very worried about it, but again, it would be nice if the commit message at least stated that this is an intentionally accepted deviance.